src/main/webapp/app/exercises/modeling/manage/modeling-exercise-detail.component.html

```diff
@@ -47,22 +43,23 @@ <h2><span jhiTranslate="artemisApp.exercise.details">Exercise Details</span></h2
                     </button>
                 </dt>
                 <dd class="editor-outline-background">
-                    <jhi-modeling-editor
-                        *ngIf="sampleSolutionUML"
-                        [readOnly]="true"
-                        [diagramType]="modelingExercise.diagramType!"
-                        [umlModel]="sampleSolutionUML!"
-                    ></jhi-modeling-editor>
+                    <jhi-modeling-editor *ngIf="sampleSolutionUML" [readOnly]="true" [diagramType]="modelingExercise.diagramType!" [umlModel]="sampleSolutionUML!">
```

Comment:
I comment it here, since it is the first changed line. If you take a look at line 35 of this file, there is the title for the sample solution declared (at least I think this is the case). Although it says "Sample Solution" within the span, the translation string resolves to "Example solution" which leads to the following inconsistency. I think this would touch some other files as well so you do not have to change it in this PR. I only add it here so we do not miss it.
![image](https://user-images.githubusercontent.com/80622272/145819094-fc6a58e2-589a-41d7-a45c-2f68305aec81.png)

Reply:
You are right, since the key name is `exercises.sampleSolution` I missed the value is actually "Example Solution". Since the surrounding context in the code explicitly refer to Modeling Exercises I think it can be a good middle ground to use `modelingExercises.sampleSolution` here and leave the actual `exercises.sampleSolution` value as-before in the scope of this PR as discussed.